velox/common/memory/MemoryAllocator.h

```diff
@@ -206,6 +206,7 @@ class MemoryAllocator : public std::enable_shared_from_this<MemoryAllocator> {
   static constexpr int32_t kMaxSizeClasses = 12;
   static constexpr uint16_t kMinAlignment = alignof(max_align_t);
   static constexpr uint16_t kMaxAlignment = 64;
+  static constexpr uint64_t kDefaultAllocatorCapacityBytes = 1L << 29; // 512 MB
```

Comment:
Why change default to 512MB but not max? Thanks!

Reply:
Yeah so this number comes from MmapAllocator. When not specifying, we create instance of MmapAllocator of 512MB. So why not set max memory for default my reasoning is by specifying max memory arbitration will never kick in and there is risk of process OOM. It might be nice to set a default value that puts any system to minimum risk.